app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -135,177 +138,70 @@ fun Home(
-
-                            Text(
-                                text = stringResource(Res.string.all_time_popular),
-                                color = MaterialTheme.colorScheme.onBackground,
-                                style = MaterialTheme.typography.titleMedium,
-                                modifier = Modifier.padding(
-                                    start = dimensionResource(Res.dimen.large_padding)
-                                )
-                            )
-
-                            Spacer(Modifier.size(dimensionResource(Res.dimen.medium_padding)))
-
-                            MediaSmallRow(
-                                mediaList = allTimePopularList,
-                                onItemClick = { itemId ->
+                            HomeRow(
+                                list = allTimePopularList,
+                                title = stringResource(Res.string.all_time_popular),
+                                onItemClicked = {
                                     navigator.navigate(
                                         MediaPageDestination(
-                                            id = itemId,
+                                            id = it.id,
                                             mediaTypeArg = homeMediaType.rawValue
                                         )
                                     ) {
                                         launchSingleTop = true
                                     }
-                                },
-                                content = { media, onItemClick ->
-                                    MediaSmall(
-                                        height = dimensionResource(Res.dimen.media_card_height),
-                                        width = dimensionResource(Res.dimen.media_card_width),
-                                        image = media?.coverImage?.extraLarge,
-                                        // TODO: Do something about this chain.
-                                        label = media?.title?.romaji ?:
-                                        media?.title?.english ?:
-                                        media?.title?.native ?: "",
-                                        onClick = {
-                                            onItemClick(media?.id)
-                                        }
-                                    )
                                 }
                             )
 
-                            Spacer(
-                                Modifier.height(
-                                    dimensionResource(Res.dimen.navigation_bar_height)
-                                            + dimensionResource(Res.dimen.large_padding)
-                                )
-                            )
+                            Spacer(Modifier.height(dimensionResource(Res.dimen.navigation_bar_height)))
```

Comment:
Bottom padding is not the same as before.

Reply:
Did I forget to move it up a level? Oop